crates/rnote-ui/src/appwindow/appsettings.rs

```diff
@@ -446,9 +443,17 @@ impl RnAppWindow {
 
         {
             // Appwindow
-            app_settings.set_int("window-width", self.width())?;
-            app_settings.set_int("window-height", self.height())?;
-            app_settings.set_boolean("is-maximized", self.is_maximized())?;
+            match self.is_maximized() {
```

Comment:
for booleans `if {..} else {..}`  is cleaner imo

Reply:
fixed